src/status/contracts.nim

```diff
@@ -1,4 +1,5 @@
-import sequtils, strformat, sugar, macros, tables, eth/common/eth_types, stew/byteutils, nimcrypto
```

Comment:
we shouldn't move this file, it's used by libstatus, and if we move it it breaks the abstraction, forces other files in `libstatus/` to `import ../contracts` etc.. , `libstatus/` should be thought of as a separate library

Reply:
My line of thinking was that these contracts were not really generic enough to not be libstatus-specific, hence the move. But i see how libstatus imports them, so they're coupled to a degree and should be considered as part of the libstatus lib.